### PR TITLE
French translation update

### PR DESCRIPTION
**Pull request by piorrro33**
100% complete. Not sure it is the most accurate, so I'll use it for now to see if some strings are inaccurate. All this was made by hand.

### CODE REVIEW

File: locales/fr_FR/pcsx2_Iconized.po [hunk 1]
```diff
@@ -180,15 +182,15 @@ msgstr ""
...
+#, c-format
 msgid ""
 "This wizard will help guide you through configuring plugins, memory cards, "
 "and BIOS.  It is recommended if this is your first time installing %s that "
 "you view the readme and configuration guide."
 msgstr ""
 "Cette aide à la configuration vous permettra de paramétrer les plugins, les "
-"cartes mémoire et le BIOS. Il est recommandé, si cela est votre premiere "
-"installation de %s, de prendre connaissance du Lisez-moi et des guides de "
+"cartes mémoire et le BIOS. Il est recommandé, si c'est votre premiere "
```
Comment (gregory38) [inline locales/fr_FR/pcsx2_Iconized.po]: accent sur premiere

File: locales/fr_FR/pcsx2_Iconized.po [hunk 2]
```diff
@@ -212,11 +214,11 @@ msgid ""
...
 msgstr ""
-"Un historique de paramètres %s a été trouvé dans le dossier de "
-"configuration. Voulez-vous que le logiciel importe ces données ou les "
-"remplace avec les valeurs par défaut de %s ?\n"
+"Des paramètres existants %s ont été trouvés dans le dossier de "
+"configuration. Voulez-vous importer ces données ou les remplacer avec les "
+"valeurs par défaut de %s ?\n"
 "\n"
-"(ou appuyez sur le bouton Annuler pour modifier le dossier d'installation)"
+"(Appuyez sur le bouton Annuler pour modifier le dossier des paramètres)"
```
Comment (gregory38) [inline locales/fr_FR/pcsx2_Iconized.po]: "pour choisir un autre dossier"

File: locales/fr_FR/pcsx2_Iconized.po [hunk 3]
```diff
@@ -647,37 +643,37 @@ msgstr ""
...
-#, fuzzy
 msgid ""
 "-2 - Reduces the EE's cyclerate by about 50%.  Moderate speedup, but *will* "
 "cause stuttering audio on many FMVs."
 msgstr ""
-"3 - Diminue l'EE Cyclerate d'environ 50% : amélioration modérée des "
-"performances, mais le son de certaines cinématiques sera insupportable."
+"-2 - Diminue la fréquence du processeur EE d'environ 50% : amélioration "
+"modérée des performances, mais le son de certaines cinématiques sera "
+"*insupportable*."
```
Comment (gregory38) [inline locales/fr_FR/pcsx2_Iconized.po]: haché ? Déformé ?

File: locales/fr_FR/pcsx2_Iconized.po [hunk 4]
```diff
@@ -736,10 +734,12 @@ msgid ""
 "bring about an increase in in-game FPS while also making games more "
 "demanding and possibly causing glitches."
 msgstr ""
-"Les valeurs définissent la vitesse de l'horlogue du CPU core R5900 de "
+"Les valeurs définissent la vitesse de l'horloge du CPU R5900 de "
 "l'EmotionEngine, et entraînent une amélioration des performances conséquente "
 "pour les jeux incapables d'utiliser tout le potentiel des composants de la "
-"Playstation 2."
+"PlayStation 2. En même temps, des valeurs plus élevées augmentent réellement "
```
Comment (gregory38) [inline locales/fr_FR/pcsx2_Iconized.po]: En meme temps => D'un autre cote?

File: locales/fr_FR/pcsx2_Main.po [hunk 5]
```diff
@@ -424,7 +422,7 @@ msgstr ""
 
 #: pcsx2/gui/AppInit.cpp:139
 msgid "PCSX2 Recompiler Error(s)"
-msgstr "Erreurs(s) du PCSX2 Recompiler"
+msgstr "Erreurs(s) du recompilateur PCSX2"
```
Comment (gregory38) [inline locales/fr_FR/pcsx2_Main.po]: Je pense que le (s) fait doublon

**Comment by piorrro33** [comment]
Yeah these are the places I didn't modify, let me correct this and make a new pull request.

**Pull request closed without merging**